package.json

```diff
@@ -14,10 +14,18 @@
     "Christopher Pappas <damassi.pappas@gmail.com>",
     "Eloy Durán <eloy.de.enige@gmail.com>"
   ],
+  "scripts": {
+    "test": "./node_modules/tape/bin/tape ./test/*.js"
```

Comment:
Small nit, but can we use Jest here to keep things in line with other artsy JS code? Should be zero setup since Jest will automatically pickup `.test.js` extensions.

Reply:
I was actually using it before but the issue is `jest` doesn't respect `node.cache`, it implements it's own cache and version of `require`

Reply:
Ahh that makes sense, np then

Reply:
Smart move